plugins/out_stackdriver/stackdriver.c

```diff
@@ -941,6 +944,102 @@ static msgpack_object *get_payload_labels(struct flb_stackdriver *ctx, msgpack_o
     return NULL;
 }
 
+/*
+ *    pack_resource_labels():
+ *  - Looks through the resource_labels parameter and appends new key value
+ *    pair to the log entry.
+ *  - Supports field access, plaintext assignment and environment variables.
+ */
+static int pack_resource_labels(struct flb_stackdriver *ctx,
```

Comment:
Not sure if this is the issue my UI but params are not indented.

Reply:
Fixed indentation.